ui/src/router/components/configuration/components/router_config_section/RulesConfigTable.js

```diff
@@ -23,16 +23,18 @@ export const RulesConfigTable = ({ routes, rules = [], defaultTrafficRule }) =>
     });
 
     // Append Default Traffic Rule
-    const defaultRuleRoutes = [];
-    defaultTrafficRule.routes.forEach((ruleRoute) => {
-        const filteredRoute = routes.find(route => route.id === ruleRoute);
-        defaultRuleRoutes.push(filteredRoute);
-    });
-    updatedRules.push({
-      name: "default-traffic-rule",
-      conditions: [],
-      routes: defaultRuleRoutes
-    });
+    if (defaultTrafficRule) {
```

Comment:
So just to confirm what was happening previously was that we've been appending a 'ghost' rule (see below) to the updated rules list, when there should be 0 rules? 

```js
{
      name: "default-traffic-rule",
      conditions: [],
      routes: [],
}
```

Reply:
Ah nope, that's actually this part here - https://github.com/caraml-dev/turing/pull/244/files#diff-4356a77431e53504f977550d197cb1c75c1754691de008de5d288d578469e5e3R115.

Reply:
Ahhh okay, in which case I've got a different comment, why don't we do `if (routes)` or `if (routes.length !== 0)` instead? 😅